styles.css

```diff
@@ -0,0 +1,100 @@
+* {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  margin:0;
+  padding:0;
+  line-height: 1.8;
+}
+
+header {
+  background-color: rgb(13,73,132);
+  color: rgb(7,196,47);
+  font-style:italic;
+}
+
+.header-container {
+  display: flex;
+  width:100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  border-color: rgb(13,73,132);
+}
+
+.header-item {
+  align-self:flex-end;
+}
+
+nav {
+  width: 100%;
+  background-color: rgb(217,217,217);
+  color:rgb(13,73,132);
+  padding:2%;
```

Comment:
Make sure you're consistent with formatting your css, you're missing spaces here and there.

```
h1 {
  color: black;
}
```